### PR TITLE
Minor fixes for cutting PyPI releases - update Makefile, common.mk, setup.py

### PR DESCRIPTION
**Pull request by chmreid**
Make modifications to setup.py and release rule in Makefile to prepare for a PyPI release.

- update make release rule to check if file exists before grepping in said file
- update author email
- update version number to 7.0.0

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -6,12 +6,12 @@
 install_requires = [line.rstrip() for line in open(os.path.join(os.path.dirname(__file__), "requirements.txt"))]
 
 setup(
-    name="dbio",
+    name="dbio-cli",
```
Comment (amarjandu) [inline setup.py:9]: dumb q?  is this gonna change the command from `dbio` to `dbio-cli`? 🤔   or is this only the pypi app?

**Review by amarjandu — APPROVED** [review]
1 dumb q

File: setup.py (hunk 1 above)
Comment (chmreid) [reply setup.py:9]: I think the actual command line utility's name is specified elsewhere, but this is a good question. Let me look into that and confirm that this won't change the CLI tool's name.

Comment (chmreid) [reply setup.py:9]: Line 27 (https://github.com/DataBiosphere/data-store-cli/pull/56/files#diff-2eeaed663bd0d25b7e608891384b7298R27) of setup.py is where the CLI tool gets its name: 

```
entry_points={
        'console_scripts': [
            'dbio=dbio.cli:main'
        ],
    },
```

There, we specify that the command `dbio` (that's where the name of the tool is specified) should map to the python file at `dbio/cli.py`, and the function `main()`.

**Pull request merged**